dump/match_helper.py

```diff
@@ -1,6 +1,23 @@
 from dump.match_infra import MatchRequest
 from dump.helper import handle_multiple_keys_matched_error
 
+# Return dict helper methods
+
+def check_status(ret):
```

Comment:
I believe this function is checking error and returns True if error. In that case check_status name sounds misleading to the functionality.
Also why do we need a separate function for this? Can you have the functionality inside get_matched_keys itself?

Reply:
I'll change the name. 

But i think it okay to have a separate function. get_matched_keys() is one API which can operate on ret[] dict, In future when we have more API's which can parse ret dict, having a separate function which can check error will be handy and thus defining these separately